test/tstDistributedTree.cpp

```diff
@@ -28,8 +28,8 @@
 
 namespace tt = boost::test_tools;
 
-using PairIndexRank = Kokkos::pair<int, int>;
-using TupleIndexRankDistance = Kokkos::pair<Kokkos::pair<int, int>, float>;
+using PairIndexRank = ArborX::PairIndexRank;
+using TupleIndexRankDistance = ArborX::Details::TupleIndexRankDistance;
```

Comment:
```suggestion
using ArborX::PairIndexRank;
using ArborX::Details::TupleIndexRankDistance;
```
or (maybe better) just qualify when you use them

Reply:
Too many places to qualify. Did the change.